CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.10
+
+### Changed
+- Uses v5.0.0 of the console-ui by default
```

Comment:
```suggestion
- Uses v5.0.0 of the filmdrop-ui by default
```

Reply:
also, there is a `console-ui` dir in this project, is that confusing?